src/read.c

```diff
@@ -1251,6 +1251,7 @@ void ReadFuncExpr (
     /* 'function( ... arg )' takes a variable number of arguments              */
     if (narg >= 1 && ! strcmp( "arg", CSTR_STRING( ELM_LIST(nams, narg) ) ) )
       {
+	/* TODO: remove this before the next non-beta release */
```

Comment:
You're right. Sorry unconnected change got caught up.

I'm not sure how common this kind of thing is. I guess extra warnings, or warnings instead of errors in the beta release might be fairly normal in which case this makes sense.